docs/sources/query/analyzer.md

```diff
@@ -235,4 +245,48 @@ level=info ts=2022-03-23T11:55:45.221254326Z caller=loki.go:355 msg="Loki starte
 
 <script src="../analyzer/script.js"> </script>
 
+## Additional Sample Queries
+
+These are some additional sample queries that you can use in the LogQL simulator.
+
+### Logfmt
+
+```logQL
+| logfmt | level = "debug"
+```
+
+Parses logfmt-formatted logs and returns only log lines where the "level" field is equal to "debug".
+
+```logQL
+| logfmt | msg="server listening on addresses"
+```
+
+Parses logfmt-formatted logs and returns only log lines with the message “server listening on address.”
+
+### JSON
+
+```logQL
+| json | level="INFO" | file="SpringApplication.java" | line_format `{{.class}}`
+```
+
+Parses JSON-formatted logs, filtering for lines where the 'level' field is "INFO" and the 'file field is "SpringApplication.java", then formats the line to return only the 'class' field.
+
+```logQL
+|~ `(T|t)omcat`
+```
+
+Performs a regular expression filter for the string 'tomcat' or 'Tomcat', without using a parser.
+
+### Unstructured text
+
+```logQL
+| pattern "<_> - <_> <_> \"<method> <url> <protocol>\" <status> <_> <_> \"<_>\" <_>" | method="GET"
+```
+
+Parses unstructured logs with the pattern parser, filtering for lines where the HTTP method is "GET".
+
+```logQL
+| pattern "<_> - <user> <_> \"<method> <url> <protocol>\" <status> <_> <_> \"<_>\" <_>" | user=~"kling.*"
+```
```

Comment:
info strings are usually expected to be lowercase

````suggestion
```logql
| logfmt | level = "debug"
```

Parses logfmt-formatted logs and returns only log lines where the "level" field is equal to "debug".

```logql
| logfmt | msg="server listening on addresses"
```

Parses logfmt-formatted logs and returns only log lines with the message “server listening on address.”

### JSON

```logql
| json | level="INFO" | file="SpringApplication.java" | line_format `{{.class}}`
```

Parses JSON-formatted logs, filtering for lines where the 'level' field is "INFO" and the 'file field is "SpringApplication.java", then formats the line to return only the 'class' field.

```logql
|~ `(T|t)omcat`
```

Performs a regular expression filter for the string 'tomcat' or 'Tomcat', without using a parser.

### Unstructured text

```logql
| pattern "<_> - <_> <_> \"<method> <url> <protocol>\" <status> <_> <_> \"<_>\" <_>" | method="GET"
```

Parses unstructured logs with the pattern parser, filtering for lines where the HTTP method is "GET".

```logql
| pattern "<_> - <user> <_> \"<method> <url> <protocol>\" <status> <_> <_> \"<_>\" <_>" | user=~"kling.*"
```
````